form.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document Form</title>
+    <link rel="stylesheet" href="css/header.css" />
+    <link rel="stylesheet" href="css/footer.css" />
+    <link rel="stylesheet" href="css/index.css" />
+  </head>
+  <body>
+    <!--Logo-->
+    <header>
+      <img
+        id="karma-logo"
+        src="./img/karma-logo.svg"
+        alt="Karma Logo"
+        width="30rem"
+        height="30rem"
+      />
+      <nav>
+        <ul class="lists">
+          <li><strong>Meet karma</strong></li>
+          <li>How it works</li>
+          <li class="store">Store</li>
```

Comment:
In this assignment, you need to make the links of "Store" and "Meet karma" active, so that when a user clicks on them, they go to the store/homepage.